products/datavirt/_common/product.yml

```diff
@@ -26,6 +26,14 @@ guides:
 description: 'A data virtualization tool useful for any organization struggling to gain operational and analytical insights from data dispersed and locked up in various business units, applications, and technology silos. (formerly known as JBoss Enterprise Data Services Platform).'
 default_download_artifact_type: installer
 downloads:
+  6.1.0.GA:
+    release_date: 2015-03-11
+    description: Minor release of JBoss DV 6.x series
```

Comment:
Kenny requested that you use the description from the Beta. So:

    Improved User Interface, OpenShift Beta Cartridge, Kerberos, Audit Log in Dashboard, MongoDB, Cloudera/Impala. Apache Solr, EAP 6.3, JDG 6.2.